packages/sit-onyx/src/components/OnyxBottomBar/OnyxBottomBar.vue

```diff
@@ -0,0 +1,70 @@
+<script lang="ts" setup>
+import { useDensity } from "../../composables/density";
+import type { OnyxBottomBarProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxBottomBarProps>(), {
+  hideTopBorder: false,
+});
+
+defineSlots<{
+  /**
+   * Bottom bar content. This slot is left aligned and should be used by buttons and icon buttons.
+   */
+  left?(): unknown;
+  /**
+   * Bottom bar content. This slot is right aligned and should be used by buttons and icon buttons.
+   */
+  right?(): unknown;
+}>();
+
+const { densityClass } = useDensity(props);
+</script>
+
+<template>
+  <div
+    :class="[
+      'onyx-component',
+      'onyx-bottom-bar',
+      densityClass,
+      { 'onyx-bottom-bar--border': !props.hideTopBorder },
+    ]"
+  >
+    <div class="onyx-bottom-bar__left">
+      <slot name="left"></slot>
+    </div>
+    <div class="onyx-bottom-bar__right">
+      <slot name="right"></slot>
+    </div>
+  </div>
+</template>
+
+<style lang="scss">
+@use "../../styles/mixins/layers.scss";
+
+.onyx-bottom-bar {
+  @include layers.component() {
+    position: fixed;
+    bottom: 0;
+    width: var(--onyx-grid-margin-inline, 100%);
```

Comment:
```suggestion
    margin-inline: var(--onyx-grid-margin-inline);
```

onyx-grid-margin-inline is responsible for centering so we should use `margin-inline` instead :)